src/Symfony/Component/VarExporter/composer.json

```diff
@@ -0,0 +1,36 @@
+{
+    "name": "symfony/var-exporter",
+    "type": "library",
+    "description": "A blend of var_export() + serialize() to turn any serializable data structure to plain PHP code",
+    "keywords": ["export", "serialize"],
```

Comment:
I'd add `symfony` and `var-exporter` as well, they are the component 💃

Reply:
this is already in the name of the component, no need to duplicate